src/common/const.ts

```diff
@@ -125,6 +125,7 @@ export const FIXED_DEVICE_CLASS_ICONS = {
   current: mdiCurrentAc,
   date: mdiCalendar,
   distance: mdiArrowLeftRight,
+  duration: mdiTimerOutline,
```

Comment:
Since we already use this for the `timer` domain, can we use a different icon? Something like a clock? Or maybe 
![CleanShot 2022-10-03 at 10 48 07](https://user-images.githubusercontent.com/5662298/193536959-e14ea45d-0380-45ba-8b34-22d7c0385f54.png)

Reply:
Yeah! Let me update my PR